packages/server/src/db/index.js

```diff
@@ -637,8 +637,7 @@ async function getGrantsInterested({ agencyId, perPage, currentPage }) {
                 .whereIn('agencies.id', agencies.map((subAgency) => subAgency.id));
         })
         .orderBy('created_at', 'DESC')
-        .offset((currentPage - 1) * perPage)
-        .limit(perPage);
+        .paginate({ currentPage, perPage, isLengthAware: true });
```

Comment:
Doing this should be equivalent to `fetchTotalRecentActivities`. We will need to add unit test coverage here and update all instances of `getGrantsInterested` to now look for `result.data` instead of the `result` itself.

Reply:
Are you planning to add test coverage in another PR before this gets merged to staging, or afterwards?

Reply:
@chrisxkeith I'm not planning to add tests at the moment. If you're good with this then the PR will get merged into your branch *not* `_staging`.

Once its in your branch it would be good if you can QA and add tests 🙏 

Does that work for you?

Reply:
I'm willing to do that, but have no idea when my dev environment will be back up. :(